src/d3/hapticplot.d3.spec.ts

```diff
@@ -93,6 +93,19 @@ describe('VR Haptic Plot', () => {
     const result = hasHaptics(controller);
     expect(result).toEqual(expectedRes);
   });
+
+  it('initilizes data points with sound components', () => {
+    hapticplot.init(scene, [10, 20, 30]);
+    const a = hasSounds(scene, shape);
+    expect(a).toEqual(true);
```

Comment:
nit: in this case you aren't gaining anything with this single letter intermediate variable;  consider replacing with 
```suggestion
    expect(hasSounds(scene, shape)).toEqual(true);
```
(same below).

more generally, variable names should tell you what they represent, and single letter variables are only typically used as iteration counters (e.g let i; i < bounds; i++); many teams opt to have their linters yell at you for single letter variables.

Reply:
fixed it!